pages/Bitcoin-Price-Predictor.py

```diff
@@ -0,0 +1,4 @@
+from page_handler import PageHandler
+
+page_handler = PageHandler("pages/pages.json")
```

Comment:
This will throw an error if you haven't added your form details to `pages/pages.json`. Kindly fix this.